libs/coin-evm/.unimportedrc.json

```diff
@@ -14,5 +14,16 @@
     "src/editTransaction/index.ts",
     "src/operation.ts"
   ],
-  "ignoreUnimported": ["src/__tests__/*", "**/*.test.*", "**/*.spec.*", "src/datasets/ethereum1.ts"]
-}
+  "ignoreUnimported": ["src/__tests__/*", "**/*.test.*", "**/*.spec.*", "src/datasets/ethereum1.ts"],
+  "ignoreUnresolved": [
+    "@ethersproject/bytes",
+    "@ethersproject/logger",
+    "@ethersproject/bignumber",
+    "@ethersproject/properties",
+    "@ethersproject/address",
+    "@ethersproject/constants",
+    "@ethersproject/strings",
+    "@ethersproject/hash",
+    "@ethersproject/keccak256"
```

Comment:
Can't pass the CI without this.

It make absolutely **ZERO** sense to have this unimported script in a library made to be used without a unique entrypoint.